src/compressed_tensors/quantization/observers/base.py

```diff
@@ -64,6 +65,34 @@ def get_qparams(
         :return: tuple of scale and zero point based on last observed value
         """
         if observed is not None:
-            # re-calcualte scale and zero point, update the stored value
-            self._scale, self._zero_point = self.calculate_qparams(observed)
+            group_size = self.quantization_args.group_size
+            if group_size is None:
+
+                # re-calcualte scale and zero point, update the stored value
+                self._scale, self._zero_point = self.calculate_qparams(observed)
+                if hasattr(self, "inc"):
+                    self.inc()
```

Comment:
why hardcode this in the parent class instead of calling after super()?

Reply:
Actually each observer may have their own unique func to run after, so i can make that instead of .inc()
for ex. Observer1 wants to do weight /= 2 or something, then instead of calling that specific func we can call